lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -37,8 +37,8 @@
 
   &:focus-within {
     .nx-radio-checkbox__control {
-      border-color: #02a6d9;
-      stroke: #02a6d9;
+      border-color: $nx-focus-border-color;
+      stroke: $nx-focus-border-color;
       filter: drop-shadow(0 0 3px rgba(2, 166, 217, 0.4));
```

Comment:
It's not written in hex format but I believe this is the same color.

Reply:
> It's not written in hex format but I believe this is the same color.

https://github.com/sonatype/sonatype-react-shared-components/pull/241/commits/3e0e378d3293dcee864b59be5b66b959b2725845